### DISCUSSION TITLE
html to handelbars

### DISCUSSION THREAD

**Discussion by rafikApp** [handlebars-lang/handlebars.js#1870] [Q&A]
Hello everyone.
Can someone convert this html code to handelbars.js code please. 

<!DOCTYPE html> 

<html>

<body>

    <script>
        function greet() {
            window.location = 'https://www.google.com';
        }
        setTimeout(greet, 2000);

    </script>

    <img src="https://drive.google.com/......" />

</body>

</html>

**Comment by satyajitnayk** [comment]
Handlebars.js handle the rendering of dynamic HTML templates.There is no direct equivalent to the setTimeout function or window redirection.
**html file**
```
<script id="greeting-template" type="text/x-handlebars-template">
    <img src="https://drive.google.com/......" />
</script>
```

**JS file**
```
$(document).ready(function() {
    setTimeout(function() {
        window.location.href = 'https://www.google.com';
    }, 2000);

    var template = Handlebars.compile($('#greeting-template').html());
    var renderedTemplate = template();
    $('body').append(renderedTemplate);
});
```
